recipe/meta.yaml

```diff
@@ -10,11 +10,11 @@ source:
   sha256: 0a04f1d7d35a06cdff506593330d4414ea242c9172498ce191f5742eac499e17
 
 build:
-  skip: true  # [win]
-  number: 0
+  skip: true  # [win or python_impl == 'pypy']
```

Comment:
Curious why Windows is skipped? Are we missing a dependency?

Reply:
That's historical, and it had more to do with MPI dependence, not Dask-MPI. I can look into it more and see if we can build in windows.

Reply:
Ok it seems like we have `mpi4py`, which would be the biggest blocker I would think.

Reply:
Trying it out with #23.